js/avatarMovement.js

```diff
@@ -0,0 +1,148 @@
+
```

Comment:
line breaks between sections of code are uneven - make sure these are consistent

Reply:
done